pkg/odo/cli/service/create.go

```diff
@@ -425,3 +473,27 @@ func getAlmExample(almExamples []map[string]interface{}, crd, operator string) (
 	}
 	return nil, errors.Errorf("Could not find example yaml definition for %q service in %q operator's definition.\nPlease provide a file containing yaml specification to start the service from operator\n", crd, operator)
 }
+
+func isCRInCluster(kind string, csvs *olmv1alpha1.ClusterServiceVersionList) (olmv1alpha1.ClusterServiceVersion, error) {
```

Comment:
Can we rename this function, because InCluster means this method should do an API call which it doesn’t.